portfolio.css

```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+  }
+
+  .btn2:disabled {
+    padding: 0;
+    width: 40px;
+    height: 40px;
+    cursor: none;
+  }
+
+  .desktop-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 300px 0;
+    gap: 48px;
+    width: 2063px;
+  }
+
+  .desktop-recent-works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 447px;
+    height: 48px;
+  }
+
+  .desktop-primary-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .desktop-card-works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 1171px;
+    height: 988px;
+  }
+
+  .desktop-card-work-row-1 {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 4px 0;
+    gap: 24px;
+  }
+
+  .desktop-card-1 {
+    background: #ebf0ee;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .desktop-image-cover {
+    width: 332px;
+    height: 300px;
+  }
+
+  .desktop-text-card-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 16px;
+    width: 300px;
+    height: 222px;
+    background: #fff;
+    border-radius: 8px;
+  }
+
+  .desktop-card-1-heading {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    margin-bottom: 0;
+  }
+
+  .desktop-list-languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    margin-top: 0;
+    gap: 4px;
+    width: 360px;
+    height: 32px;
+  }
+
+  .desktop-list-languages > li {
+    padding: 8px 12px;
+    list-style-type: none;
+    background: #ebf0ee;
+    border-radius: 4px;
+  }
+
+  .desktop-button-frame {
+    width: 100px;
+    height: 69px;
+  }
+
+  .desktop-btn-green {
+    color: white;
+    width: 122px;
+    height: 48px;
+    background: #36b37f;
+    border-radius: 4px;
+  }
+
+  .desktop-btn-green:hover {
+    border: 1px solid blue;
+    text-decoration: underline;
+    text-decoration-color: blue;
+  }
+
+  .desktop-btn-green:active {
+    background-color: green;
+  }
+
+  .desktop-card-work-row-2 {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 4px 0;
+    gap: 24px;
+  }
+
+  .desktop-text-card-2 {
+    display: flex;
+    flex-direction: column;
+    width: 300px;
+    height: 222px;
+    align-items: center;
+    padding: 16px;
+    gap: 16px;
+    background: #fff;
+    border-radius: 8px;
+  }
+
+  .desktop-about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 80px 0;
+    gap: 48px;
+    width: 2063px;
```

Comment:
- [ ] Kindly note that `_**About me**_` section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized just as the previous reviewer suggested 😄 

![image](https://user-images.githubusercontent.com/63473584/187044415-d2e32ef5-a3fe-47e3-bd5f-bde4f9dcffb0.png)